README.md

```diff
@@ -6,5 +6,17 @@ Fork and clone this repository.
 
 ### Questions
 1. Explain why Rails exists and what it's used for.
+
+Rails is a framework for building websites. As such, Rails establishes conventions for easier collaboration and maintenance. These conventions are codified as the Rails API
```

Comment:
I would say "web _applications_" rather than "websites", but otherwise I'd say that this is right.